swift/Samples/TicTacToe/Sources/Authentication/LoginScreen.swift

```diff
@@ -24,33 +24,23 @@ struct LoginScreen: Screen {
     var password: String
     var onPasswordChanged: (String) -> Void
     var onLoginTapped: () -> Void
-}
-
-
-extension ViewRegistry {
 
-    public mutating func registerLoginScreen() {
-        self.register(screenViewControllerType: LoginViewController.self)
+    var viewControllerDescription: ViewControllerDescription {
+        return ViewControllerDescription(
+            build: { LoginViewController() },
+            update: { $0.update(with: self) })
     }
-
 }
 
 
-fileprivate final class LoginViewController: ScreenViewController<LoginScreen> {
-    let welcomeLabel: UILabel
-    let emailField: UITextField
-    let passwordField: UITextField
-    let button: UIButton
-
-    required init(screen: LoginScreen, viewRegistry: ViewRegistry) {
-        welcomeLabel = UILabel(frame: .zero)
-        emailField = UITextField(frame: .zero)
-        passwordField = UITextField(frame: .zero)
-        button = UIButton(frame: .zero)
-        super.init(screen: screen, viewRegistry: viewRegistry)
-
-        update(with: screen)
-    }
+fileprivate final class LoginViewController: UIViewController {
```

Comment:
Made this one not be a subclass of `ScreenViewController` just to show that is’s not necessary as a superclass